Algorithms/Searching/binarysearch.cpp

```diff
@@ -27,13 +27,18 @@ int binarySearch(int arr[], int l, int r, int x)
     // not present 
     return -1; 
 } 
+  // Sample input:
```

Comment:
push the sample input output on top under the description

Reply:
Done.